docs/base-account/reference/core/getKeypair.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: "getKeypair"
+description: "Retrieve an existing P256 key pair from storage"
+---
+
+Defined in the [Base Account SDK](https://github.com/base/account-sdk)
+
+<Info>
+Retrieves an existing P256 key pair if one has been previously generated and stored. This is useful for checking if keys already exist before generating new ones.
+</Info>
+
+## Parameters
+
+This function takes no parameters.
+
+## Returns
+
+<ResponseField name="result" type="P256KeyPair | null">
+The stored P256 key pair or `null` if no key pair exists.
+
+<Expandable title="P256KeyPair properties">
+<ResponseField name="publicKey" type="string">
+The public key for the stored pair in hexadecimal format.
+</ResponseField>
+
+<ResponseField name="privateKey" type="string">
+The private key for the stored pair. Handle with extreme care.
+</ResponseField>
+</Expandable>
+</ResponseField>
+
+<RequestExample>
+```typescript Basic Usage
+import { getKeypair } from '@base-org/account';
+
+const existingKeyPair = await getKeypair();
+if (existingKeyPair) {
+  console.log('Found existing key pair');
+} else {
+  console.log('No existing key pair found');
+}
+```
+
+```typescript Get or Create Pattern
+import { getKeypair, generateKeyPair } from '@base-org/account';
+
+let keyPair = await getKeypair();
+if (!keyPair) {
+  keyPair = await generateKeyPair();
+}
+```
+</RequestExample>
+
+<ResponseExample>
+```typescript Success Response (Key Pair Found)
+{
+  publicKey: "0x04a1b2c3d4e5f6...",
+  privateKey: "0x1a2b3c4d5e6f7a..."
+}
+```
+
+```typescript Success Response (No Key Pair)
+null
+```
+</ResponseExample>
+
+## Error Handling
+
+| Code | Message | Description |
+| ---- | ------- | ----------- |
+| 4100 | Storage access denied | Cannot access secure key storage |
+| 4200 | Storage corruption | Stored key data is corrupted or invalid |
+| 4300 | Browser compatibility | Browser does not support required storage APIs |
+
+<Warning>
+**Private Key Access**
+
+The retrieved private keys should be handled with the same security considerations as newly generated keys.
+</Warning>
+
+## Get or Create Pattern
+
+A common pattern is to check for existing keys before generating new ones:
+
+```typescript
+import { getKeypair, generateKeyPair } from '@base-org/account';
+
+async function getOrCreateKeyPair() {
+  // Try to get existing key pair first
+  let keyPair = await getKeypair();
+  
+  if (!keyPair) {
+    // Generate new key pair if none exists
+    console.log('No existing key pair, generating new one...');
+    keyPair = await generateKeyPair();
+  } else {
+    console.log('Using existing key pair');
+  }
+  
+  return keyPair;
+}
+```
+
+## Integration with Account Management
```

Comment:
I would reframe this section as maybe "usage example". I don't want to confuse devs that this is how we want them to do base account account management, since this is not what this library is used for (it's for managing sub account local signers)